### DISCUSSION TITLE
Create a Project

### DISCUSSION THREAD

**Discussion by cperaltab** [USF-IMARS/mbon_zooplankton_to_dwc#15] [Ideas]
Hi! I think we could create a Project for this repo? I was trying to do so, but I couldn't find how to create it. I like the idea of having a board with the tasks we need to work on and mostly I like the Done section of the board 😊 Together with the Discussion section I think it will be very useful. 

@7yl4r can you please teach me how to create a project for other repo where I'm a contributor but not the owner? Or maybe this is not possible without me being the owner?

**Comment by 7yl4r** [comment]
I would like to meet and work through this either this Thursday or next week. Please check [my availability here](https://bit.ly/ty-calendar) and suggest a meet time for this.

**Reply by cperaltab** [reply]
Thank you Tylar. This Thursday is not possible for me. I could make it this Friday 10am or next week on Tuesday 10am. Does this works for you?

**Reply by 7yl4r** [reply]
Friday 10am works. See you then!
